.github/workflows/tests.yml

```diff
@@ -1,9 +1,8 @@
 name: tests
 on:
   push:
-    branches-ignore:
-      - 'master'
-      - 'actions'
+    branches: [ feature/python3_11_support_issue_#185 ]
```

Comment:
Undo this change. no longer needed.

Reply:
Done in - [80883e9](https://github.com/easysnmp/easysnmp/pull/186/commits/80883e9252851b14f8fbf6b1a3bfc9d342049212)